charmcraft.yaml

```diff
@@ -13,5 +13,5 @@ parts:
       - libssl-dev
       - rustc
       - cargo
-    charm-binary-python-packages: [psycopg2-binary==2.9.3]
+    charm-binary-python-packages: [psycopg2-binary==2.9.5]
```

Comment:
We need to figure out how to add/build binaries that work to remove this. Using other `parts` plugins seems like an option, but it will take time to figure out and we'll need to verify that such a custom build will keep on being supported.

Reply:
Can SNAP magically build it for us from "psycopg2"?
Drawback: PR build time will increase.

Another option is [apt package](https://packages.ubuntu.com/lunar/python3-psycopg2) or build it from sources in fat-SNAP.

Reply:
AFAIK we build snaps for the charm's workload while here we need the binary in the charm itself, so for K8S at least I doubt it can help us. The deb package we need is `libpq5`, but I couldn't figure out a way to pull it at runtime using `charmcraft.yaml` and the `charm` parts plugin. Another option we can explore is to use a pure Python postgresql client in the charm, but either way we'll need to spend some time figuring out what the best approach would be and implement it.

Reply:
I tried the apt package in the past, but some errors happened when I deployed the charm. Some more investigation is needed to figure out how to make it work or to find another option, like Dragomir commented.